src/styles/style-helpers.ts

```diff
@@ -25,6 +25,16 @@ function isTheme(args: ThemeArgs): args is Theme {
 export const getTheme = (args: ThemeArgs): Theme =>
   isTheme(args) ? args : args.theme;
 
+type StyleFn =
+  | ((theme: Theme) => SerializedStyles)
+  | ((args: ThemeArgs) => SerializedStyles)
+  | false
+  | null
+  | undefined;
```

Comment:
Just curious, why do we need these?

Reply:
Yeah this looks interesting... 🤔 🙂 

It's `| ((theme: Theme) => SerializedStyles)` or `| ((args: ThemeArgs) => SerializedStyles)`... ?

Reply:
I want to allow for falsy values so that styles can be conditionally applied with ease: 

```tsx
<div css={cx(baseStyles, condition && conditionalStyles)} />
```

The `conditionalStyles` are only applied when the `condition` is `true`.

Reply:
@hleote Are you asking about the syntax? The pipe (`|`) indeed means "or" in TypeScript.